tests/searchSpec.js

```diff
@@ -195,6 +195,39 @@ describe('search', function() {
     done();
   });
 
+  it('makes search with non existing filter value with conjunction true should return no results', function test(done) {
+
+    const itemsjs = require('./../index')(items, configuration);
+    
+    const result = itemsjs.search({
+      filters: {
+        category: ['drama', 'thriller']
+      }
+    });
+    
+    assert.equal(result.data.items.length, 0);
+    assert.equal(result.data.aggregations.tags.buckets[0].doc_count, 0);
+    
+    done();
+  });
+  
+  it('makes search with non existing filter value with conjunction false should return results', function test(done) {
+    
+    configuration.aggregations.category.conjunction = false;
+    const itemsjs = require('./../index')(items, configuration);
+
+    const result = itemsjs.search({
+      filters: {
+        category: ['drama', 'thriller']
```

Comment:
Could you introduce another similar test with category (conjunction: false)? This test should attempt to find a non-existent single value (e.g., 'thriller'). The expected outcome should be no results.

```js
const result = itemsjs.search({
  filters: {
    category: ['thriller']
  }
});
```

Reply:
added in 4840314447a7647abf23935392b3a71b7f285314 as `makes search with non existing single filter value with conjunction false should return no results`